README.md

```diff
@@ -22,8 +29,16 @@ def largest?(array, value)
   return true
 end
 ```
+#--------------------#
+
+Snippet 2 - Big O: O(n * m)
```

Comment:
Yes, please! I feel like I haven't truly differentiated the various new concepts in my head yet. 
https://www.youtube.com/watch?v=es2T6KY45cA I liked these video but they never mentioned what type of big O notation it was)